schemas/com.mbta.ctd.glides.trips_updated.v1.json

```diff
@@ -1,91 +1,33 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://github.com/mbta/schemas/blob/main/schemas/glides-events-schema.json",
-  "title": "Glides Events",
-  "description": "Kinesis events emitted by Glides",
-  "$ref": "cloudevents.json",
+  "$id": "https://github.com/mbta/schemas/blob/main/schemas/com.mbta.ctd.glides.trips_updated.v1.json",
+  "description": "Trips Updated",
   "type": "object",
   "properties": {
-    "type": {
-      "type": "string",
-      "description": "name of the event",
-      "$comment": "enforced to be a valid event name by the oneOf, below"
-    },
+    "type": { "const": "com.mbta.ctd.glides.trips_updated.v1" },
     "specversion": { "const": "1.0" },
-    "source": { "const": "glides.mbta.com" },
-    "id": { "type": "string", "minLength": 1 },
-    "time": { "$ref": "#/$defs/timestamp" },
+    "source": { "$ref": "glides-events.json#/properties/source" },
+    "id": { "$ref": "glides-events.json#/properties/id" },
+    "time": { "$ref": "glides-events.json#/$defs/timestamp" },
     "data": {
       "type": "object",
       "properties": {
-        "metadata": { "$ref": "#/$defs/metadata" }
-      },
-      "required": ["metadata"],
-      "$comment": "other fields validated by the oneOf, below"
-    }
-  },
-  "required": ["type", "specversion", "source", "id", "time", "data"],
-  "oneOf": [
-    {
-      "properties": {
-        "type": { "const": "com.mbta.ctd.glides.editors_changed.v1" },
-        "data": { "$ref": "#/$defs/editors_changed_v1" }
-      }
-    },
-    {
-      "properties": {
-        "type": { "const": "com.mbta.ctd.glides.operator_signed_in.v1" },
-        "data": { "$ref": "#/$defs/operator_signed_in_v1" }
-      }
-    },
-    {
-      "properties": {
-        "type": { "const": "com.mbta.ctd.glides.trips_updated.v1" },
-        "data": { "$ref": "#/$defs/trips_updated_v1" }
-      }
-    }
-  ],
-  "$defs": {
-    "editors_changed_v1": {
-      "type": "object",
-      "properties": {
-        "changes": {
-          "type": "array",
-          "contains": { "$ref": "#/$defs/editor_change" }
-        }
-      },
-      "required": ["changes"]
-    },
-    "operator_signed_in_v1": {
-      "type": "object",
-      "properties": {
-        "operator": { "$ref": "#/$defs/operator" },
-        "signedInAt": { "$ref": "#/$defs/timestamp" },
-        "signature": { "$ref": "#/$defs/signature" }
-      },
-      "required": []
-    },
-    "trips_updated_v1": {
-      "type": "object",
-      "properties": {
+        "metadata": { "$ref": "glides-events.json#/$defs/metadata" },
         "tripUpdates": {
           "type": "array",
-          "contains": {
+          "items": {
```

Comment:
I think this is the correct change, but something else is resulting in the sub-objects not appearing in the rendered schema:
<img width="719" alt="Screenshot 2023-06-14 at 3 09 29 PM" src="https://github.com/mbta/schemas/assets/214921/6ce8ec5e-a8d9-40b6-b16c-41bf4860eb28">